allennlp_models/common/model_card.py

```diff
@@ -42,6 +58,60 @@ def __str__(self):
 class ModelDetails(ModelCardInfo):
     """
     This provides the basic information about the model.
+
+    # Parameters:
+
+    description: `str`
+        A high-level overview of the model.
+        Eg. The model implements a reading comprehension model patterned
+            after the proposed model in [Devlin et al, 2018]
+            (https://api.semanticscholar.org/CorpusID:52967399), with improvements
+            borrowed from the SQuAD model in the transformers project.
+            It predicts start tokens and end tokens with a linear layer on top of
+            word piece embeddings.
+
+    developed_by: `str`
+        Person/organization that developed the model. This can be used by all
+        stakeholders to infer details pertaining to model development and
+        potential conflicts of interest.
+
+    contributed_by: `str`
+        Person that contributed the model to the repository.
+
+    date: `str`
+        The date on which the model was contributed. This is useful for all
+        stakeholders to become further informed on what techniques and
+        data sources were likely to be available during model development.
+        Format example: 2020-09-23
+
+    version: `str`
+        The version of the model, and how it differs from previous versions.
+        This is useful for all stakeholders to track whether the model is the
+        latest version, associate known bugs to the correct model versions,
+        and aid in model comparisons.
+
+    model_type: `str`
+        The type of the model; the basic architecture. This is likely to be
+        particularly relevant for software and model developers, as well as
+        individuals knowledgeable about machine learning, to highlight what
+        kinds of assumptions are encoded in the system.
+        Eg. Naive Bayes Classifier.
+
+    paper: `str`
+        The paper on which the model is based.
+        Format example:
+        [Model Cards for Model Reporting (Mitchell et al, 2019)]
+        (https://api.semanticscholar.org/CorpusID:52946140)
+
+    citation: `str`
+        The BibTex for the paper.
```

Comment:
Is it useful to have that in here? Semantic Scholar already has a link for that.